README.md

```diff
@@ -71,7 +92,7 @@ final HCaptchaConfig config = HCaptchaConfig.builder()
                 .resetOnTimeout(true)
                 .theme(HCaptchaTheme.DARK)
                 .build();
-HCaptcha.getClient(this).verifyWithHCaptcha(config)...;
+HCaptcha.getClient(this, config).verifyWithHCaptcha()...;
```

Comment:
This would be a breaking change. 

Can we somehow maintain the old api and add a new parameter to the `HCaptchaConfig.lazy = true/false` which would basically just not `fragment.show()` but require an explicit `.show()` to be called to show the hCaptcha view. Thus, the new api is not a breaking change and allows for both old and new lazy approach.

 What do you reckon @CAMOBAP @e271828- ?

Example:

```java
final HCaptchaConfig config = HCaptchaConfig.builder()
                ...
                .theme(HCaptchaTheme.DARK)
                .lazy(true)
                .build();
final HCaptcha hCaptchaClient = HCaptcha.getClient(this).verifyWithHCaptcha(config);

// Async when ready to show, client would do:
hCaptchaClient.show()
  .addOnSuccessListener(new OnSuccessListener<HCaptchaTokenResponse>() {
        @Override
        public void onSuccess(HCaptchaTokenResponse response) {
            String userResponseToken = response.getTokenResult();
            // Validate the user response token using the hCAPTCHA siteverify API.
        }
    })
    .addOnFailureListener(new OnFailureListener() {
        @Override
        public void onFailure(HCaptchaException e) {
            Log.d("MainActivity", "Error code: " + e.getStatusCode());
            Log.d("MainActivity", "Error msg: " + e.getMessage());
        }
    });
```

Reply:
This seems like a better approach, yeah.

Reply:
> This would be a breaking change.

Definitely

> What do you reckon @CAMOBAP @e271828- ?

In this case, API becomes less clear for me because depends on `lazy` param `verifyWithHCaptcha` will work significantly different, + extra API method `show`

So between `less clear API` vs `breaking change` I would select the first one

Reply:
Looks great, will update PR soon

Reply:
Done